e2e_tests/tests/cluster/test_job_queue.py

```diff
@@ -80,3 +81,22 @@ def get_job_weight(self, jobID: str) -> str:
                 continue
             return value_dict["Weight"]
         return ""
+
+
+@pytest.mark.e2e_cpu
+def test_job_queue_is_working_fine() -> None:
```

Comment:
I think a manual test is fine since we are just removing code. Pretty sure this is covered by some other e2es, too.

Reply:
Removed the test case and added screenshots for manual test results.